string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,14 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  let bool = true;
+
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] === word[i].toLowerCase()) {
+      continue;
+    } else {
+      bool = false;
+      break;
+    }
+  }
+  return bool;
+}
```

Comment:
We can actually simplify this one a lot! We don't even need a `for` loop. We can do something like this & just compare the original word to the lowerCased version
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```